custom_components/grocy/__init__.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import os
 from datetime import timedelta
+import hashlib
```

Comment:
could you run `isort` over all files to ensure proper imports?

Reply:
Lol I didn't it was important. Thx